src/content/1.7/code/agda/snippet12.agda

```diff
@@ -0,0 +1,9 @@
+instance
+  floatEq : Eq Float
+  floatEq ._==_ = λ x y → does (x ≟ y)
```

Comment:
floatEq seems like something that could live only in Section107.agda and skip here